### ISSUE TITLE
Use alternative font on non-Windows OS

### ISSUE THREAD

**Issue by Symbian9** [OpenOrienteering/mapper#739]
In most cases Linux distributives not include Arial, because its proprietary commercial font.

So, propose make Mapper functionality for automatically detect if Arial presented in OS and if not, then try use nearest free alternatives (that in most cases already installed in Linux)
* https://en.wikipedia.org/wiki/Arial#Free_alternatives

Another way is creating tab in settings dialog like in LibreOffice
* https://help.libreoffice.org/Common/Fonts#Replacement_table

REFERENCES:
* http://typedia.com/blog/post/cure-for-the-common-webfont-part-1-alternatives-to-arial-and-helvetica/
* http://practicaltypography.com/helvetica-and-arial-alternatives.html
* http://practice.typekit.com/reference/superior-alternative-to-arial/

**Comment by dg0yt** [comment]
Or https://benperove.com/install-microsoft-truetype-fonts-in-fedora-and-ubuntu/

**Comment by Symbian9** [comment]
@dg0yt, I don't think that is fully legal, but anyway Mapper at least must give a warning message if in openned map used fonts, that not installed and ask what fonts use as alternative.

**Comment by Abbe98** [comment]
It's legal to install the fonts but it's not legal to bundle them within other software.

I guess it would be enough to just provide a hint to the solution by @dg0yt in the manual?
